backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ParameterMembershipImpl.java

```diff
@@ -74,6 +74,14 @@ public Feature basicGetOwnedMemberParameter() {
                 .findFirst()
                 .orElse(null);
     }
+    
+    /**
+     * ownedMemberParameter : Feature {redefines ownedMemberFeature}
```

Comment:
Please add `@generated NOT` in every modified method in the generated code of the metamodel.
Please check every other modified method in this PR.